lib/winpki/ldap.go

```diff
@@ -392,8 +392,10 @@ func (c *LDAPConfig) createConnection(ctx context.Context, ldapTLSConfig *tls.Co
 		)
 
 		if err != nil {
-			// If the connection fails, try the next server
-			c.Logger.DebugContext(ctx, "Error connecting to LDAP server, trying next available server", "server", server, "error", err)
+			if c.LocateServer.Enabled {
```

Comment:
This entire block is already inside a conditional checking if `LocateServer` is enabled, so I don't think you need it here.